tests/functional/vagrant/dump.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${0}")" && pwd)"
+cd "${SCRIPT_DIR}"
+
+set +e
+set -x
+for hn in storage0 storage1 storage2 storage3; do
+    echo '+++' $hn
+    vagrant ssh $hn -- rpm -qa | grep gluster
+    vagrant ssh $hn -- systemctl status gluster-blockd
```

Comment:
It will be good if we can add tcmu-runner and targetcli as well.

Reply:
I don't see a strong need to do that as of yet. Let's prove that the current approach adds value first.

Reply:
cool no problem